Hulpe Dalia Miruna/3.8L5/main.c

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+//I will use the symbols for Roman numerals: I,V,X,L,C,D,M;
+//In this way, the biggest number that can be obtain would be: 3999;
+//The longest length for the string would be 15 characters: number (MMMDCCCLXXXVIII->3888)
+
+int main()
+{   int i,sign,ok,j,p,k,NumericValue[7]={1,5,10,50,100,500,1000},NewNumber=0;
```

Comment:
Format your code and extract things to functions - way too much stuff happening in Main